<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TreeWidget</class>
 <widget class="QMainWindow" name="TreeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TreeWidget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>817</width>
         <height>505</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menucatalogue_tree">
    <property name="title">
     <string>catalogue tree</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionTraverse"/>
    <addaction name="actionAddDocument"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuview">
    <property name="title">
     <string>view</string>
    </property>
    <addaction name="actionMagnify"/>
    <addaction name="actionMinify"/>
    <addaction name="actionVisiable"/>
    <addaction name="separator"/>
    <addaction name="actionHeight"/>
    <addaction name="actionWidth"/>
    <addaction name="separator"/>
    <addaction name="actionVisible"/>
   </widget>
   <addaction name="menucatalogue_tree"/>
   <addaction name="menuview"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd"/>
   <addaction name="actionAddDocument"/>
   <addaction name="actionDelete"/>
   <addaction name="actionTraverse"/>
   <addaction name="separator"/>
   <addaction name="actionMagnify"/>
   <addaction name="actionMinify"/>
   <addaction name="actionVisiable"/>
   <addaction name="separator"/>
   <addaction name="actionHeight"/>
   <addaction name="actionWidth"/>
   <addaction name="separator"/>
   <addaction name="actionFloat"/>
   <addaction name="actionVisible"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="showDropIndicator" stdset="0">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="headerCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="headerMinimumSectionSize">
        <number>60</number>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>150</number>
       </attribute>
       <attribute name="headerHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="headerShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <attribute name="headerStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>node</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="background">
         <color>
          <red>85</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </property>
       </column>
       <column>
        <property name="text">
         <string>node type</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="background">
         <color>
          <red>255</red>
          <green>255</green>
          <blue>0</blue>
         </color>
        </property>
       </column>
       <item>
        <property name="text">
         <string>picture</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icons/icons/open3.bmp</normaloff>:/icons/icons/open3.bmp</iconset>
        </property>
        <property name="text">
         <string>picture</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
        <item>
         <property name="text">
          <string>New Item</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/icons/icons/15.ico</normaloff>:/icons/icons/15.ico</iconset>
         </property>
         <property name="text">
          <string/>
         </property>
         <item>
          <property name="text">
           <string>picture1</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icons/icons/31.ico</normaloff>:/icons/icons/31.ico</iconset>
          </property>
          <property name="text">
           <string/>
          </property>
         </item>
        </item>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/folder1.bmp</normaloff>:/icons/icons/folder1.bmp</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionAddDocument">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/824.bmp</normaloff>:/icons/icons/824.bmp</iconset>
   </property>
   <property name="text">
    <string>AddDocument</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/delete1.bmp</normaloff>:/icons/icons/delete1.bmp</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionTraverse">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/fold.bmp</normaloff>:/icons/icons/fold.bmp</iconset>
   </property>
   <property name="text">
    <string>Traverse</string>
   </property>
  </action>
  <action name="actionMagnify">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/418.bmp</normaloff>:/icons/icons/418.bmp</iconset>
   </property>
   <property name="text">
    <string>Magnify</string>
   </property>
  </action>
  <action name="actionMinify">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/416.bmp</normaloff>:/icons/icons/416.bmp</iconset>
   </property>
   <property name="text">
    <string>Minify</string>
   </property>
  </action>
  <action name="actionHeight">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/422.bmp</normaloff>:/icons/icons/422.bmp</iconset>
   </property>
   <property name="text">
    <string>Height</string>
   </property>
  </action>
  <action name="actionWidth">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/424.bmp</normaloff>:/icons/icons/424.bmp</iconset>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
  </action>
  <action name="actionFloat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/814.bmp</normaloff>:/icons/icons/814.bmp</iconset>
   </property>
   <property name="text">
    <string>Float</string>
   </property>
  </action>
  <action name="actionVisiable">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/414.bmp</normaloff>:/icons/icons/414.bmp</iconset>
   </property>
   <property name="text">
    <string>Actual</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/exit.bmp</normaloff>:/icons/icons/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionVisible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/preview1.bmp</normaloff>:/icons/icons/preview1.bmp</iconset>
   </property>
   <property name="text">
    <string>Visible</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>treeWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
